addons/mail/static/src/components/follower/follower.scss

```diff
@@ -2,55 +2,26 @@
 // Layout
 // ------------------------------------------------------------------
 
-.o_Follower {
-    display: flex;
-    flex-flow: row;
-    justify-content: space-between;
-    padding: map-get($spacers, 0);
-}
-
 .o_Follower_avatar {
-    width: 24px;
-    height: 24px;
-    margin-inline-end: map-get($spacers, 2);
+    width: $o-mail-partner-avatar-size;
+    height: $o-mail-partner-avatar-size;
 }
 
 .o_Follower_details {
-    align-items: center;
-    display: flex;
-    flex: 1;
+    flex-basis: 0;
```

Comment:
Did you have a case that this rule is relevant?

The container with list of follower is so minuscule in the UI, that this rule looks like it can make a bit sense on tiny UI, when the truncate is so heavy that it's a benefit to show avatar + follower name on 2 different lines to see part of name (and also considering that the avatar is not distinct enough, that showing a bit of follower name helps).

Just wondering whether you see some reasons to add? This looks so niche and a change that is out of scope of PR.

Reply:
Indeed, after doing more tests, it seems that there is no reason to add it.
Because `flex: 1` is the short version of `flex: 1 1 0%`, I thought there was a reason to keep a `flex-basis: 0` but finally it seems not 😄 .

I removed it 👍